CHANGES.rst

```diff
@@ -87,6 +87,8 @@ Cubeviz
 - Fixes Spectral Extraction's assumptions of one data per viewer, and flux data only in
   flux-viewer/uncertainty data only in uncert-viewer. [#2646]
 
+- Fixed incorrect output units when Velocity is selected in Moment Maps plugin. [#2665]
```

Comment:
Maybe this entry needs to be more descriptive: Which moment was wrong and how.

Reply:
I just realized this doesn't need a separate changelog entry, it's a fix to a new 3.9 feature.

Reply:
Oh, phew!!!

Reply:
You might still want to attach this PR number to the original entry.